lib/node/node.js

```diff
@@ -228,13 +234,13 @@ module.exports.create = function createNode(type, expectedParams, options) {
     BaseNode.prototype.validateAndBuild = function() {
         this.json.type = this.type;
         Object.keys(expectedParams).forEach(function(expectedParamName) {
-            if (!this.params.hasOwnProperty(expectedParamName)) {
-                throw new Error('Missing required param "' + expectedParamName + '"');
-            }
-
             var param = this.params[expectedParamName];
             var validator = expectedParams[expectedParamName];
 
+            if (!validator.nullable && !this.params.hasOwnProperty(expectedParamName)) {
```

Comment:
Oh!. That was the place. When I tried it, I put this check at the wrong place and I did that the implementation was awkward.